src/engine/bank04.asm

```diff
@@ -1073,7 +1073,68 @@ OverworldScriptTable: ; 1217b (4:617b)
 	dw ScriptCommand_EndScriptLoop9
 	dw ScriptCommand_EndScriptLoop10
 
-	INCROM $1224b, $1229f
+
+MultichoiceTextbox_ConfigTable_ChooseDeckToDuelAgainst: ;1224b
+	db $04, $00 	; x, y to start drawing box
+	db $10, $08 	; width, height of box
+	db $06, $02 	; x, y coordinate to start printing next text
+	tx Text03f6 	; text id to print next
+	db $06, $04 	; x, y coordinate to start printing next text
+	tx Text03f7 	; text id to print next
+	db $06, $06 	; x, y coordinate to start printing next text
+	tx Text03f8 	; text id to print next 
+	db $ff 			; marker byte -- end text entries
+	db $05, $02 	; cursor starting x, y
+	db $02 			; number of tiles the cursor moves per toggle
+	db $03 			; cursor max index
+	db $0f 			; curor image
```

Comment:
A note on style, we typically use tabs only at the beginning of the line for instruction alignment, and use spaces for comment alignment - because tabs are sometimes rendered with different widths (like here on GitHub) which messes with alignment.

Reply:
Fixed in the most recent commit